recipe/meta.yaml

```diff
@@ -12,13 +12,13 @@ source:
 build:
   skip: True # [py<37]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - meson 1.0.1
+    - meson >=0.63.3
     - pip
-    - pyproject-metadata 0.7.1
+    - pyproject-metadata >=0.7.1
     - python
     - setuptools
     - tomli 1.2.2  # [py<311]
```

Comment:
```suggestion
    - tomli >=1.0.0  # [py<311]
```

Reply:
the run requirement needs a py<311 selector too

Reply:
addressed